.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+```
+
+Also note the class name convention. The method `getWarningList()` MUST return an array of line numbers indicating warnings (when running `phpcs`) found in `WPThemeReview/Tests/CoreFunctionality/FileIncludeSniff.inc`.
+If you run:
+
+```bash
+$ cd /path-to-cloned/WPThemeReview
+$ vendor/bin/phpcs --standard=WPThemeReview -s WPThemeReview/Tests/CoreFunctionality/FileIncludeUnitTest.inc --sniffs=WPThemeReview.CoreFunctionality.FileInclude
+...
+FILE: /Users/infinum-denis/Projects/Personal/WPThemeReview/WPThemeReview/Tests/CoreFunctionality/FileIncludeUnitTest.inc
+--------------------------------------------------------------------------------
+FOUND 0 ERRORS AND 6 WARNINGS AFFECTING 6 LINES
+--------------------------------------------------------------------------------
+ 3 | WARNING | Check that include is not being used to load template
+   |         | files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 4 | WARNING | Check that include_once is not being used to load
+   |         | template files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 5 | WARNING | Check that require is not being used to load template
+   |         | files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 6 | WARNING | Check that require_once is not being used to load
+   |         | template files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 7 | WARNING | Check that include is not being used to load template
+   |         | files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 8 | WARNING | Check that require_once is not being used to load
+   |         | template files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+--------------------------------------------------------------------------------
+
+Time: 45ms; Memory: 6Mb
+```
+You'll see the line number and number of WARNINGs we need to return in the `getWarningList()` method.
+
+The `--sniffs=...` directive limits the output to the sniff you are testing.
+
+You can also run the same command using composer scripts
+
+```bash
+composer check-cs -- --standard=WPThemeReview -s WPThemeReview/Tests/CoreFunctionality/FileIncludeUnitTest.inc --sniffs=WPThemeReview.CoreFunctionality.FileInclude
+```
+
+## Code Standards for this project
+
+The sniffs and test files - not test _case_ files! - for WPCS should be written such that they pass the `WordPress-Extra` and the `WordPress-Docs` code standards using the custom ruleset as found in `/.phpcs.xml.dist`.
```

Comment:
This needs to be adjusted to a more WPTRCS specific text + could mention that to test this you can run the `composer check-cs` command from the project root.